content/04.methods.md

```diff
@@ -156,6 +156,7 @@ The top 50 principal components were selected and used to calculate UMAP embeddi
 
 If any libraries included in the ScPCA project contain additional ADT data, the ADT data are also merged and stored in the `altExp` slot of the merged `SingleCellExperiment` object.
 By contrast, if any libraries included in the ScPCA project are multiplexed and contain HTO data, no merged object is created.
+Merged objects were not created for any projects with more than 50 samples due to size constraints.
```

Comment:
```suggestion
Merged objects were not created for projects with more than 50 samples because of the computational resources that would be required for working with those objects.
```